notifications_utils/version_tools/requirements_for_test_common.in

```
@@ -1,12 +1,11 @@
-beautifulsoup4==4.11.1
-pytest==7.2.0
-pytest-env==0.8.1
-pytest-mock==3.9.0
-pytest-xdist==3.0.2
-pytest-testmon==2.1.0
-pytest-watch==4.2.0
-requests-mock==1.10.0
-freezegun==1.2.2
+beautifulsoup4==4.12.3
+pytest==8.3.4
+pytest-env==1.1.5
+pytest-mock==3.14.0
+pytest-xdist==3.6.1
+pytest-testmon==2.1.1
+requests-mock==1.12.1
+freezegun==1.5.1
 
 black==24.4.0  # Also update `.pre-commit-config.yaml` if this changes
 ruff==0.3.7  # Also update `.pre-commit-config.yaml` if this changes
```

These also seem to work with the existing API/admin code (you’ll also need to update `.pre-commit-config.yaml`):
```suggestion
black==24.10.0  # Also update `.pre-commit-config.yaml` if this changes
ruff==0.8.2  # Also update `.pre-commit-config.yaml` if this changes
```

---

yeah i was wondering about if i should do that separately but lets Just Do It